src/workerd/api/http.h

```diff
@@ -757,6 +762,8 @@ class Response: public Body {
   jsg::WontImplement getUseFinalUrl() { return jsg::WontImplement(); }
   // This is deprecated in the spec.
 
+  bool canUseWebSocketCompression() { return hasEnabledWebSocketCompression; }
```

Comment:
It looks like this is only called from one place, and that place is a method of this class, so that one place could just access `hasEnabledWebSocketCompression`  directly and this method could be removed?